grails-app/services/org/pih/warehouse/data/LoadDataService.groovy

```diff
@@ -200,52 +209,120 @@ class LoadDataService {
         csvReader.close();
     }
 
+    private Transaction createAdjustmentTransaction(Date transactionDate, Inventory inventory) {
+        return new Transaction(
+                transactionDate: transactionDate,
+                transactionType: TransactionType.get(Constants.ADJUSTMENT_CREDIT_TRANSACTION_TYPE_ID),
+                inventory: inventory
+        )
+    }
+
+    private TransactionEntry createAdjustmentTransactionEntry(
+            Map<String, String> csvReaderRow,
+            Product product,
+            Location binLocation,
+            Map<String, AvailableItem> availableItems
+    ) {
+        DateFormat dateFormat = new SimpleDateFormat('DD/mm/yyyy')
```

Comment:
Shouldn't it be `'dd/MM/yyyy'`?

"D" is day in year and "m" is month in hour: https://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html

Actually, why isn't it our normal expiration date format? From Constants.groovy:
```
EXPIRATION_DATE_FORMAT = "MM/dd/yyyy"
```

I know this isn't new code but I don't understand how it was ever working correctly...

Reply:
yeah it'd be good to take a look while you're already modifying this code. If it's importing expiry dates totally wrong, it'd probably be an easy fix